command/server/config.go

```diff
@@ -77,6 +77,9 @@ type Config struct {
 	EnableResponseHeaderHostname    bool        `hcl:"-"`
 	EnableResponseHeaderHostnameRaw interface{} `hcl:"enable_response_header_hostname"`
 
+	EnableLogRequests    bool        `hcl:"-"`
```

Comment:
Instead of a boolean, what do you think about something more general?  For example, we could make this be a string argument that specifies the log level at which to log requests.  Or we could make it be something like `log_request_info = "basic"`, which would leave us room to extend this feature later without needing a new config option.

Reply:
Sounds great! I will enable logging in all levels which will depend on this configuration. 
So, do you mean that `basic` is going to be the default value for this variable? and in which log level should `basic` be resolved?